tools/meteor-services/deploy.js

```diff
@@ -321,6 +323,206 @@ function canonicalizeSite(site) {
   return parsed.hostname;
 };
 
+// Executes the poll to check for deployment success and outputs proper messages
+// to user about the status of their app during the polling process
+function executePollForDeploymentSuccess(versionId, deployPollTimeout, result) {
+  // Create a default polling configuration for polling for deploy / build
+  // In the future, we may change this to be user-configurable or smart
+  // The user can only currently configure the polling timeout via a flag
+  let pollingConfiguration = new PollingConfiguration(deployPollTimeout);
+  
+  var deploymentPollResult = buildmessage.enterJob({
+    title: "Waiting for Deploy to succeed..."}, 
+    function() {
+      return pollForDeploy(pollingConfiguration,
+        versionId,
+        true);
+    });
+
+  if(deploymentPollResult && deploymentPollResult.isActive) {
+    // The deploy succeeded and the deployed version is now active
+    const {newVersionNum, hostname, username, region, galaxyUrl} = result.payload;
+    var successfulDeploymentMessage = `
```

Comment:
Hmm. So the reason we send down the message from the server is that it allows us to improve the message just by changing galaxy-server (immediately affecting even users of relatively old `meteor`). eg we were able to add the warning about `meteor.com` when people kept getting confused that we didn't support it any more.

I'm not saying it's a bad idea to have more structured data in the result and to have some of the message be calculated here but at the very least it would be good to have some field that the server can set for "extra messages".